sql: TestSqlActivityUpdateTopLimitJob failed

sql.TestSqlActivityUpdateTopLimitJob [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10815485?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10815485?buildTab=artifacts#/) on release-23.1 @ [3eeb4361863fd8ad291a635676b2adf44b6acb4d](https://github.com/cockroachdb/cockroach/commits/3eeb4361863fd8ad291a635676b2adf44b6acb4d):


```
=== RUN   TestSqlActivityUpdateTopLimitJob
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/b287de7194e17a0386ee1775f0d198ba/logTestSqlActivityUpdateTopLimitJob1111712255
    test_log_scope.go:79: use -show-logs to present logs inline
    sql_activity_update_job_test.go:304: condition failed to evaluate within 30s: Need to wait for row to populate with follower_read_timestamp.
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/b287de7194e17a0386ee1775f0d198ba/logTestSqlActivityUpdateTopLimitJob1111712255
--- FAIL: TestSqlActivityUpdateTopLimitJob (39.99s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSqlActivityUpdateTopLimitJob.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-29541

--- a/pkg/sql/sql_activity_update_job_test.go
+++ b/pkg/sql/sql_activity_update_job_test.go
@@ -332,7 +332,7 @@ func TestSqlActivityUpdateTopLimitJob(t *testing.T) {
 			}
 			require.Equal(t, stmtAggTs, txnAggTs)
 			return nil
-		}, 30*time.Second)
+		}, 1*time.Minute)
 
 		// The max number of queries is number of top columns * max number of
 		// queries per a column (6*3=18 for this test, 6*500=3000 default). Most of

